src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,135 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Physical type and encoding for the geometry type.
+ */
+enum GeometryEncoding {
```

Comment:
Why is this an enum rather than being a requirement? This seems like we're being too generic to me.

Reply:
I don't think that anything should be here unless it has a clear and necessary use right now. We can always add new enums later, and we can also add new optional fields in the logical type struct. I would cut this to keep the spec simple.

Reply:
`GeometryEncoding` is not something that we have to set for now as we have only one option. Perhaps we can simply add a comment to GEOMETRY type to enforce WKB. Later on an optional `enum GeometryEncoding` can be added once new encoding has been proposed officially.

Reply:
Readers not checking if a GeometryEncoding is present will then still read wrong data in that scenario.

To me, it feels safer to simply keep this enum.

Reply:
To workaround the legacy reader, we have to add separate geometry type if a new encoding has been introduced. From the discussion so far, it seems that we'd better keep `enum GeometryEncoding`.

Reply:
I think that if this is going to be present, then those future concerns about how it will be evolved must be defined. Should readers fail if there is an unsupported value?